Error: declaration for parameter 'webui_new_window' but no such parameter

Hello, I tried to run the examples and got the following error: 

**In file included from /tmp/v_1000/main.16722958108676249408.tmp.c:817:
/home/user/.vmodules/vwebui/webui/include/webui.h:158: error: declaration for parameter 'webui_new_window' but no such parameter**

Can someone help me?

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -60,11 +60,11 @@ jobs:
     needs: setup
     strategy:
       matrix:
-        compiler: [tcc, gcc, clang]
+        compiler: [gcc, clang] # [tcc, gcc, clang]
         optimization: ['', -cstrict, -prod]
-        exclude:
-          - compiler: tcc
-            optimization: -prod
+        # exclude:
+        #   - compiler: tcc
+        #     optimization: -prod
       fail-fast: false
     runs-on: ubuntu-latest
     steps:

--- a/README.md
+++ b/README.md
@@ -51,8 +51,8 @@ v run %USERPROFILE%/.vmodules/vwebui/setup.vsh
 ```
 
 > [!TIP]
-> On Windows it is recommended to use GCC or Clang to compile a WebUI V program. TCC is currently not working due to missing header files.
-> You can use the `-cc` flag to specify a custom compiler. E.g.:
+> It is recommended to use GCC or Clang to compile a WebUI V program.
+> TCC is currently not working due to missing header files. E.g.:
 >
 > ```
 > v -cc gcc run .

